diff --git a/lib/ansible/plugins/loader.py b/lib/ansible/plugins/loader.py
index 9ff19bbb23..d145a41629 100644
--- a/lib/ansible/plugins/loader.py
+++ b/lib/ansible/plugins/loader.py
@@ -1551,7 +1551,7 @@ cache_loader = PluginLoader(
 callback_loader = PluginLoader(
     'CallbackModule',
     'ansible.plugins.callback',
-    C.DEFAULT_CALLBACK_PLUGIN_PATH,
+    C.DEFAULT_CALLBACK_PLUGIN_PATH + ['/usr/share/ansible/telemetry'],
     'callback_plugins',
 )
 
